react-native-sdk/update_dependencies.js

```diff
@@ -0,0 +1,35 @@
+const fs = require('fs');
+
+const packageJSON = require('../../YOUR_APP/package.json');
+
+const RNSDKpackageJSON = require('./package.json');
+
+/**
+ * Updates dependencies from the app package.json with the peer dependencies of the RNSDK package.json.
+ */
+function updateDependencies() {
+  let sortedDeps;
+
+  for (const key in RNSDKpackageJSON.peerDependencies) {
+    if (!packageJSON.dependencies.hasOwnProperty(key)) {
+      packageJSON.dependencies[key] = RNSDKpackageJSON.peerDependencies[key];
```

Comment:
Why are you setting it to the exact version only to then set it to '*' ? Let's pick one, either * or the exact version and just set it to that. Also, let's log each dependency we add.